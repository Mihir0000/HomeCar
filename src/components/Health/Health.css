.health_section {
    width: 98%;
    float: left;
    padding-bottom: 60px;
    margin-bottom: 50px;
}

.layout_padding {
    padding-top: 90px;
    padding-bottom: 0px;
}

.header_section {
    width: 100%;
    float: left;
    background-image: url(../../../public/static/img/banner-bg.png);
    height: auto;
    background-size: 100%;
    padding: 10px 0px 25px 0px;
    background-repeat: no-repeat;
}

.health_taital {
    width: 100%;
    float: left;
    font-size: 40px;
    color: #252525;
    font-weight: bold;
}

.health_text {
    width: 100%;
    float: left;
    font-size: 16px;
    color: #252525;
    margin: 0px;
}

.image_main {
    width: 100%;
    position: relative;
}

.image_main:after {
    content: '';
    position: absolute;
    background-color: #cfedf0;
    height: 450px;
    width: 300px;
    left: 0px;
    right: initial;
    z-index: -1;
    margin: 0 auto;
    top: -40px;
}

.image_main:hover .image {
    background-color: rgba(27, 236, 222, 0.5);
}

.image_main:hover .middle {
    opacity: 1;
}
.image {
    opacity: 1;
    display: block;
    width: 100%;
    height: auto;
    transition: .5s ease;
    backface-visibility: hidden;
}
.middle {
    transition: .5s ease;
    opacity: 0;
    position: absolute;
    top: 50%;
    left: 57%;
    transform: translate(-50%, -50%);
    -ms-transform: translate(-50%, -50%);
    text-align: center;
}

.main {
    width: 100%;
    margin: 40px 0px 0px 40px;
    box-shadow: 0px 0px 14px 2px #f3f3f3;
    position: relative;
}

.main::after {
    content: '';
    position: absolute;
    background-color: rgba(27, 236, 222, 0.5);
    margin: 0 auto;
    left: 0px;
    right: 0px;
    top: 0px;
    width: 100%;
    height: 100%;
    display: none;
}

.main:hover::after {
    background-color: rgba(27, 236, 222, 0.5);
    display: block;
}


.text {
    background-color: #252525;
    color: white;
    font-size: 16px;
    padding: 32px 32px;
    border-radius: 100px;
}

.image_main_1 {
    width: 100%;
    position: relative;
}

.image_main_1:hover .image {
    background-color: rgba(27, 236, 222, 0.5);
}

.image_main_1:hover .middle {
    opacity: 1;
}




.getquote_bt_1 {
    width: 180px;
    float: left;
    margin-top: 100px;
}

.getquote_bt_1 a {
    width: 100%;
    float: left;
    font-size: 18px;
    color: #ffffff;
    background-color: #1becde;
    padding: 7px 0px;
    text-align: center;
}

.getquote_bt_1 a:hover {
    background-color: #252525;
}

