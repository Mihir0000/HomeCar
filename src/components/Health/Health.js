import React from 'react';
import { Card, Col, Container, Image, Row } from 'react-bootstrap';
import './Health.css';

const Health = ({ text }) => {
    return (
        <Container fluid className="health_section layout_padding">
            <Container>
                <Card className="bg-transparent border-0">
                    <Card.Title className="health_taital fs-1">
                        {text}
                    </Card.Title>
                    <Card.Text className="health_text">
                        Lorem ipsum dolor sit amet, consectetur adipiscing elit,
                        sed do eiusmod tempor incididunt ut labore et dolore
                        magna aliqua. Ut enim ad minim veniam, quis
                    </Card.Text>
                </Card>
                <Card className="health_section layout_padding bg-transparent border-0">
                    <Row>
                        <Col sm={7}>
                            <Card.Body className="image_main">
                                <Card.Title className="main">
                                    <Image
                                        src="static/img/img-2.png"
                                        alt="Avatar"
                                        className="image"
                                        style={{ width: '100%' }}
                                    />
                                </Card.Title>
                                <Card.Title className="middle">
                                    <Card.Subtitle className="text">
                                        <Image
                                            src="static/img/icon-1.png"
                                            style={{ width: '40px' }}
                                            alt="icon1"
                                        />
                                    </Card.Subtitle>
                                </Card.Title>
                            </Card.Body>
                        </Col>
                        <Col sm={5}>
                            <Card.Body className="image_main_1">
                                <Card.Title className="main">
                                    <Image
                                        src="static/img/img-3.png"
                                        alt="Avatar"
                                        className="image"
                                        style={{ width: '100%' }}
                                    />
                                </Card.Title>
                                <Card.Title className="middle">
                                    <Card.Subtitle className="text">
                                        <Image
                                            src="static/img/icon-1.png"
                                            style={{ width: '40px' }}
                                            alt="icon1"
                                        />
                                    </Card.Subtitle>
                                </Card.Title>
                            </Card.Body>
                        </Col>
                    </Row>
                    <Card.Body className="getquote_bt_1">
                        <Card.Link
                            href="/"
                            className="d-flex justify-content-around"
                        >
                            Read More
                            <Card.Text className="d-flex align-self-center m-2">
                                <Image
                                    src="static/img/right-arrow.png"
                                    alt="rightarrow"
                                />
                            </Card.Text>
                        </Card.Link>
                    </Card.Body>
                </Card>
            </Container>
        </Container>
    );
};

export default Health;
