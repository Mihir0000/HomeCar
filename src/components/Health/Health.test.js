import React from 'react';
import { render } from '@testing-library/react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Health from './Health';

test('snapshot testing --Health', () => {
    const component = renderer.create(<Health />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

// crash test --Health
it('render without crashing --health', () => {
    const div = document.createElement('div');
    render(<Health />, div);
    ReactDOM.unmountComponentAtNode(div);
});
