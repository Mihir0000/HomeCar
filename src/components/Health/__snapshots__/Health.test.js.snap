// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing --Health 1`] = `
<div
  className="health_section layout_padding container-fluid"
>
  <div
    className="container"
  >
    <div
      className="bg-transparent border-0 card"
    >
      <div
        className="health_taital fs-1 card-title h5"
      />
      <p
        className="health_text card-text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
      </p>
    </div>
    <div
      className="health_section layout_padding bg-transparent border-0 card"
    >
      <div
        className="row"
      >
        <div
          className="col-sm-7"
        >
          <div
            className="image_main card-body"
          >
            <div
              className="main card-title h5"
            >
              <img
                alt="Avatar"
                className="image"
                src="static/img/img-2.png"
                style={
                  Object {
                    "width": "100%",
                  }
                }
              />
            </div>
            <div
              className="middle card-title h5"
            >
              <div
                className="text card-subtitle h6"
              >
                <img
                  alt="icon1"
                  className=""
                  src="static/img/icon-1.png"
                  style={
                    Object {
                      "width": "40px",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-sm-5"
        >
          <div
            className="image_main_1 card-body"
          >
            <div
              className="main card-title h5"
            >
              <img
                alt="Avatar"
                className="image"
                src="static/img/img-3.png"
                style={
                  Object {
                    "width": "100%",
                  }
                }
              />
            </div>
            <div
              className="middle card-title h5"
            >
              <div
                className="text card-subtitle h6"
              >
                <img
                  alt="icon1"
                  className=""
                  src="static/img/icon-1.png"
                  style={
                    Object {
                      "width": "40px",
                    }
                  }
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="getquote_bt_1 card-body"
      >
        <a
          className="d-flex justify-content-around card-link"
          href="/"
        >
          Read More
          <p
            className="d-flex align-self-center m-2 card-text"
          >
            <img
              alt="rightarrow"
              className=""
              src="static/img/right-arrow.png"
            />
          </p>
        </a>
      </div>
    </div>
  </div>
</div>
`;
